source/conf.py

```diff
@@ -42,7 +42,7 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
-html_title = 'Qogecoin docs'
```

Comment:
I think all the files are generated because this makes global changes. Could you try to send this line as a separate commit?

Reply:
Actually never mind i will fix this line later. Just send it without this line then changes should be smaller.

Reply:
Alright.